db/models/topics.model.js

```diff
@@ -1,7 +1,16 @@
 const db = require("../../db/connection");
 
-exports.selectAllTopics = () => {
-  return db.query(`SELECT * FROM topics`).then((results) => {
+exports.selectAllTopics = (topic) => {
+  const validTopics = ["mitch", "cats", "paper"];
```

Comment:
this way of handling things works but could be problematic down the line if you made a post topics for instance

Reply:
You could and should query the DB for the topic and then act accordingly here, instead of hard coding the valid topics.